### PR TITLE
Bump Android X

### LINKED ISSUE #13843: android.content.res.Resources$NotFoundException: Unable to find resource ID #0xffffffff
### NOTES

The issue here appears to be that google removed some resources/apis that we are compiling against.  It looks like we will need to start pinning the AndroidX dlls or compile against newer releases of AndroidX at a faster cadence 

### Original ticket

I updated my project this morning with some new android files via newget and ever since, when I try to navigate using the flyout menu, I get the following error:  android.content.res.Resources$NotFoundException: Unable to find resource ID #0xffffffff

### Description
The files I updated that caused this issue were:

Xamarin.AndroidX.MediaRouter 1.2.1 -> 1.2.2.1
Xamarin.AndroidX.LeanBack 1.0.0.6 -> 1.0.0.7
Xamarin.AndroidX.Palette 1.0.0.6 -> 1.0.0.7

when I rolled the files back I no longer have the issue

### Steps to Reproduce

1. Update your Xamarin app to the latest android files
2. Try to use flyout navigation 
3.  I know one of the files I updated was Xamarin.AndroidX.Palette

### Basic Information

- Version with issue:
- Last known good version:
- Platform Target Frameworks: 
  - iOS:  
  - Android:  
  - UWP:   
- Android Support Library / AndroidX Version: 
- NuGet Packages:
- Affected Devices:

### Environment
Microsoft Visual Studio Professional 2019 Version 16.8.6
Xamarin Forms 5.0.0.2012

<details><summary>Show/Hide Visual Studio info</summary>

```
```

</details>

### PR DESCRIPTION
**Pull request by PureWeen**
### Description of Change ###

Bump to latest Android X nugets

### Issues Resolved ### 
- fixes #13843

### Platforms Affected ### 
- Android

### PR Checklist ###

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

**Comment by PureWeen** [comment]
API 21 is failing `ShellItemItemsClearTests_google_pixel_2_8_1_0`

**Comment by Tommigun1980** [comment]
Will this ever be merged? Android release builds are broken right now as linking fails.

"Mono.Cecil.ResolutionException: Failed to resolve AndroidX.Core.View.Accessibiity.AccessibilityNodeInfoCompat" <-- notice the misspelling.

https://github.com/xamarin/Xamarin.Forms/issues/13969 was closed but it appears as this hasn't been merged yet so I don't think it's fixed.

Also, imagine new people starting to use XF and builds just won't work for them, wouldn't it make sense to merge this in soon and make a new release? I think that the ability to produce builds is an important feature.

Thank you.

**Comment by jsuarezruiz** [comment]
This PR will be included in the next Release.

**Comment by Tommigun1980** [comment]
> This PR will be included in the next Release.

Fantastic, thank you!!

**Comment by jamesmontemagno** [comment]
Can we align this with the Community Toolkit & Xamarin.Essentials.

**Comment by PureWeen** [comment]
> Can we align this with the Community Toolkit & Xamarin.Essentials.

@mattleibow @jfversluis @jsuarezruiz   ?

FYI this PR is pinning AndroidX upper versions

File: .nuspec/Xamarin.Forms.nuspec [hunk 1]
```diff
@@ -15,10 +15,17 @@
...
     <dependencies>
       <group targetFramework="MonoAndroid10.0">
-        <dependency id="Xamarin.AndroidX.Lifecycle.LiveData" version="2.2.0.4" />
-        <dependency id="Xamarin.Google.Android.Material" version="1.2.1.1" />
-        <dependency id="Xamarin.AndroidX.Legacy.Support.V4" version="1.0.0.6" />
-        <dependency id="Xamarin.AndroidX.Browser" version="1.3.0" />
+        <dependency id="Xamarin.AndroidX.Lifecycle.LiveData" version="[2.2.0.4,2.3)" />
+        <dependency id="Xamarin.Google.Android.Material" version="[1.2.1.1,1.3)" />
+        <dependency id="Xamarin.AndroidX.Legacy.Support.V4" version="[1.0.0.7,1.1)" />
+        <dependency id="Xamarin.AndroidX.Browser" version="[1.3.0,1.4)" />
```
Comment (jamesmontemagno) [inline .nuspec/Xamarin.Forms.nuspec:21]:
```suggestion
        <dependency id="Xamarin.AndroidX.Browser" version="[1.3.0.5,1.4)" />
```

Comment (PureWeen) [reply .nuspec/Xamarin.Forms.nuspec]: Updated

**Review by jamesmontemagno — APPROVED** [review]

**Comment by MagicAndre1981** [comment]
```
Warning	NU1608	Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2083 requires Xamarin.AndroidX.RecyclerView (>= 1.1.0.8 && < 1.2.0) but version Xamarin.AndroidX.RecyclerView 1.2.1 was resolved.	
Warning	NU1608	Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2083 requires Xamarin.Google.Android.Material (>= 1.2.1.1 && < 1.3.0) but version Xamarin.Google.Android.Material 1.4.0 was resolved.	
Warning	NU1608	Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2083 requires Xamarin.AndroidX.Lifecycle.LiveData (>= 2.2.0.4 && < 2.3.0) but version Xamarin.AndroidX.Lifecycle.LiveData 2.3.1 was resolved.
Warning	NU1608	Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2083 requires Xamarin.AndroidX.Core (>= 1.3.2.3 && < 1.4.0) but version Xamarin.AndroidX.Core 1.6.0 was resolved.
```

Why are the dependencies now restricted in Xamarin.Forms 5.0.0.2083?

**Comment by Tommigun1980** [comment]
> ```
> Warning	NU1608	Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2083 requires Xamarin.AndroidX.RecyclerView (>= 1.1.0.8 && < 1.2.0) but version Xamarin.AndroidX.RecyclerView 1.2.1 was resolved.	
> Warning	NU1608	Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2083 requires Xamarin.Google.Android.Material (>= 1.2.1.1 && < 1.3.0) but version Xamarin.Google.Android.Material 1.4.0 was resolved.	
> Warning	NU1608	Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2083 requires Xamarin.AndroidX.Lifecycle.LiveData (>= 2.2.0.4 && < 2.3.0) but version Xamarin.AndroidX.Lifecycle.LiveData 2.3.1 was resolved.
> Warning	NU1608	Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2083 requires Xamarin.AndroidX.Core (>= 1.3.2.3 && < 1.4.0) but version Xamarin.AndroidX.Core 1.6.0 was resolved.
> ```
> 
> 
>      
>   
> 
> Why are the dependencies now restricted in Xamarin.Forms 5.0.0.2083?

The commit that introduced that says just “fix versioning”: https://github.com/xamarin/Xamarin.Forms/pull/14101/commits/fbb37d9c3b89889f3f015aa853943af699830f3b

Maybe create a bug report for better visibility?

**Comment by AndreasReitberger** [comment]
> ```
> Warning	NU1608	Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2083 requires Xamarin.AndroidX.RecyclerView (>= 1.1.0.8 && < 1.2.0) but version Xamarin.AndroidX.RecyclerView 1.2.1 was resolved.	
> Warning	NU1608	Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2083 requires Xamarin.Google.Android.Material (>= 1.2.1.1 && < 1.3.0) but version Xamarin.Google.Android.Material 1.4.0 was resolved.	
> Warning	NU1608	Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2083 requires Xamarin.AndroidX.Lifecycle.LiveData (>= 2.2.0.4 && < 2.3.0) but version Xamarin.AndroidX.Lifecycle.LiveData 2.3.1 was resolved.
> Warning	NU1608	Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2083 requires Xamarin.AndroidX.Core (>= 1.3.2.3 && < 1.4.0) but version Xamarin.AndroidX.Core 1.6.0 was resolved.
> ```
> 
> Why are the dependencies now restricted in Xamarin.Forms 5.0.0.2083?

I created an issue for this. 
https://github.com/xamarin/Xamarin.Forms/issues/14417

**Pull request merged**